add reset functions to typical transactions and typical reimbursement

CoderStellaJ opened the pull request.

kangyeelim commented:
> All the tests passed after I added in the methods - resetTransaction and resetReimbursements! Maybe you can fix the checkstyle errors then merge it, so that that I can pull it?
> 
> Thank you!!

That's great! I think we should all just make it a habit to call it in our tests!

CoderStellaJ commented:
> All the tests passed after I added in the methods - resetTransaction and resetReimbursements! Maybe you can fix the checkstyle errors then merge it, so that that I can pull it?
> 
> Thank you!!

I've fixed the error. Glad that this method works!

CoderStellaJ commented:
> > All the tests passed after I added in the methods - resetTransaction and resetReimbursements! Maybe you can fix the checkstyle errors then merge it, so that that I can pull it?
> > Thank you!!
> 
> That's great! I think we should all just make it a habit to call it in our tests!

I've modified class TypicalReimbursement and changed all the objects to private non-static. 
I also removed transactions for reimbursements from your TypicalTransactions, idk whether this will affect the person test
